grype/db/curator.go

```diff
@@ -55,15 +58,23 @@ func NewCurator(cfg Config) (Curator, error) {
 	dbDir := path.Join(cfg.DBRootDir, strconv.Itoa(vulnerability.SchemaVersion))
 
 	fs := afero.NewOsFs()
-	httpClient, err := defaultHTTPClient(fs, cfg.CACert)
+	listingClient, err := defaultHTTPClient(fs, cfg.CACert)
```

Comment:
It would be nice to have better testing on this logic, which suggests there could be some refactoring that would help here.